src/components/ChatEntry.js

```diff
@@ -1,22 +1,34 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import React from 'react';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, setLikeStatus }) => {
```

Comment:
I'm just curious as to why you hardcoded these prop parameters instead of reading them in as props.id or props.sender, etc. Especially to keep things consistent.